core/templates/dev/head/pages/exploration_player/PredictionAlgorithmRegistryService.js

```diff
@@ -28,24 +28,29 @@
      *   }
      * }
      */
-     var algorithmIdPredictionServiceMapping = {};
+    var algorithmIdPredictionServiceMapping = {
+      CodeClassifier: {
+        1: 'CodeReplPredictionService'
```

Comment:
The fact that the key here is a number actually throws me off a bit. What do you think about using something like "v1" instead?